node-repository/src/main/java/com/yahoo/vespa/hosted/provision/maintenance/NodeRepositoryMaintenance.java

```diff
@@ -178,14 +180,14 @@ public void deconstruct() {
             nodeMetricsCollectionInterval = Duration.ofMinutes(1);
             operatorChangeRedeployInterval = Duration.ofMinutes(1);
             osUpgradeActivatorInterval = zone.system().isCd() ? Duration.ofSeconds(30) : Duration.ofMinutes(5);
-            periodicRedeployInterval = Duration.ofMinutes(30);
+            periodicRedeployInterval = deploymentExistsOnAllConfigServers ? Duration.ofMinutes(90) : Duration.ofMinutes(30);
```

Comment:
`PeriodicApplicationMaintainer` runs every minute (`redeployMaintainerInterval`), this is the `minTimeBetweenRedeployments`, so I don't think we should change this.

Reply:
Ouch, yes, thanks